src/views/pacbio/PacbioPoolCreate.vue

```diff
@@ -1,39 +1,36 @@
 <template>
   <data-fetcher :fetcher="fetchPoolsData">
     <div class="flex flex-col pt-4">
-      <div class="flex flex-col w-1/2 px-4">
-        <traction-menu :border="true">
-          <traction-menu-item
-            v-for="(tabTitle, index) in tabTitles"
-            :key="index"
-            :active="index == sourceIndex"
-            color="blue"
-            @click="setSource(index)"
-            >{{ tabTitle }}</traction-menu-item
-          >
-        </traction-menu>
-      </div>
       <div class="w-full grid grid-cols-2 gap-x-2 mt-4">
-        <div v-if="sourceIndex == 0" class="flex flex-col">
-          <traction-section title="Plates" number="1a">
-            <div class="text-left">Find Plates</div>
-            <LabwareFinder :fetcher="findPacbioPlate" filter="barcode" />
-          </traction-section>
-        </div>
-        <div v-else>
-          <traction-section title="Tubes" number="2a">
-            <div class="text-left">Find Tubes</div>
-            <LabwareFinder :fetcher="findPacbioTube" filter="barcode" />
+        <div class="flex flex-col">
+          <traction-section
+            title="Scan labware"
+            number="1a"
+            description="To get started, scan or type a plate, tube, or library barcode and press Enter"
```

Comment:
Should be keep library barcode in description or wait until we fully add it? Don't want to confuse users.

Reply:
Good finding! We don't need it now, as it will confuse user 👍